src/main/java/com/portatlas/response/RootResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.portatlas.response;
+
+import java.util.ArrayList;
+
+public class RootResponse {
```

Comment:
Yeah, setting the links for resources could be fore other responses as well so I should take that out and not make it specific to a root request.